backend/pkg/rest.utils/error.go

```diff
@@ -22,3 +27,19 @@ type Error struct {
 func (err Error) Error() string {
 	return err.Err
 }
+
+func IsApiError(e error) bool {
+	_, ok := e.(*Error)
+	return ok
```

Comment:
`errors.As` might be more stable as it will recursively check wrapped errors if it's the right type.
```suggestion
	var target *Error
	return errors.As(e, &target)
```